proto/token.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "permissions.proto";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "momento.auth";
+
+package token;
+
+service Token {
+  rpc GenerateTempAuthToken (_GenerateTempAuthTokenRequest) returns (stream _GenerateTempAuthTokenResponse) {}
+}
+
+message _GenerateTempAuthTokenRequest {
```

Comment:
This still needs to be finalized alternative suggestions
- GenerateEphemeralAuthToken
- GenerateProvisionalAuthToken
- GenerateBriefAuthToken
- GeneratePerishableAuthToken
- GenerateDecayingAuthToken
- GenerateTransientAuthToken

Reply:
For better or worse, we have landed on `GenerateAuthAcorn`.